recipe/conda_build_config.yaml

```
@@ -125,17 +125,24 @@ channel_sources:
 channel_targets:
   - conda-forge main
 
-cdt_name:  # [linux]
-  - cos7   # [linux]
+cdt_name:       # [linux]
+  - cos7        # [linux and os.environ.get("DEFAULT_LINUX_VERSION", "cos7") == "cos7"]
+  - conda_2_28  # [linux and os.environ.get("DEFAULT_LINUX_VERSION", "cos7") == "conda_2_28"]
```

The new CDTs do not contain the image version anymore, see [here](https://github.com/conda-forge/conda-forge.github.io/blob/5e57e30a11e91dc127aab1543b2cffe8ed8b3290/community/minutes/2024-05-15.md?plain=1#L74-L77) (as pointed out in https://github.com/conda-forge/cdt-builds/pull/70).
```suggestion
cdt_name:       # [linux]
  - cos7        # [linux and os.environ.get("DEFAULT_LINUX_VERSION", "cos7") == "cos7"]
  - conda       # [linux and os.environ.get("DEFAULT_LINUX_VERSION", "cos7") == "alma8"]
```

---

Thanks Axel! 🙏

Addressed in recent commits. Please take another look